users/views.py

```diff
@@ -56,10 +59,39 @@ def UserCreateConfirmView(request, token):
         return redirect('users:signup')
 
 
+def UserLoginView(request):
+    if request.method == "GET":
+        template = loader.get_template('users/user_login.html')
+        return HttpResponse(template.render(request=request))
+
+    if request.method == "POST":
+        email = request.POST['email']
+        password = request.POST['password']
+
+        user = authenticate(email=email, password=password)
+
+        if user is not None:
+            if user.is_active:
+                login(request, user)
+                return redirect('users:profile', pk=user.pk)
+            else:
+                messages.error(request, 'Your account is inactive.')
```

Comment:
Maybe a message that the user should click the mailed confirmation link would be useful.